package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "sib-api-typescript-node-v3-sdk",
+    "version": "0.0.1",
+    "description": "SendinBlue's API v3 TypeScript Node Library",
+    "main": "api.js",
+    "types": "api.d.ts",
+    "scripts": {
+        "clean": "rm -Rf node_modules/ *.js",
+        "build": "tsc",
+        "test": "npm run build && node client.js"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/sendinblue/APIv3-typescript-library.git"
+    },
+    "author": "Deepanshu Singh",
```

Comment:
Should be similar to https://github.com/sendinblue/APIv3-nodejs-library/blob/master/package.json#L17